packages/table/src/headers/rowHeader.tsx

```diff
@@ -188,18 +209,45 @@ export class RowHeader extends React.Component<IRowHeaderProps, {}> {
                 </Resizable>
             </DragSelectable>
         );
+
+        return (!isReorderable) ? children : (
+            <DragReorderable
+                key={Classes.rowIndexClass(rowIndex)}
+                locateClick={this.locateClick}
+                locateDrag={this.locateDragForReordering}
+                onReorder={onReorder}
+                onReorderPreview={onReorderPreview}
+                onSelection={onSelection}
+                selectedRegions={selectedRegions}
+                toRegion={this.toRegion}
```

Comment:
@giladgray actually, just reran things. Using `Regions.row` directly messes up the `this` binding within it, and changing to fat-arrow notation isn't an option for static functions. Easiest to just leave as is.